src/utilities/APIHandler/mockFetcher.js

```diff
@@ -7,6 +7,11 @@ class MockFetcher {
         const resolvedData = this.data.error ? this.data : this.data.data;
         return Promise.resolve(resolvedData);
     }
+
+    post = async (endpoint, data) => {
+        const resolvedData = this.data.error ? this.data : this.data.data;
```

Comment:
Where is this `data` name coming from? It's kinda strange to read `data.data`

Reply:
I believe Response returns everything under a `data`, and then our API wraps the specific returned data in another `data` key.
<img width="374" alt="Screen Shot 2019-07-29 at 11 16 10 AM" src="https://user-images.githubusercontent.com/29721784/62071915-5c48fd00-b1f2-11e9-9998-1f465eb430ef.png">